play.pokemonshowdown.com/src/panel-teambuilder-team.tsx

```diff
@@ -137,6 +177,20 @@ class TeamPanel extends PSRoomPanel<TeamRoom> {
 					</p>
 				</div>
 			)}
+			<br />
+			<button class="button exportbutton" onClick={this.uploadTeam} disabled={this.exported}>
+				<i class="fa fa-upload"></i> Upload to Showdown database (saves across devices)
+			</button>
+			<label>
```

Comment:
```suggestion
			<label class="label" style="display: inline-block">
```

Reply:
are we no longer using `style={{ display: 'inline-block' }}`? or was that only ever a server thing